core/quorum_recv.go

```diff
@@ -209,6 +209,18 @@ func (c *Core) quorumRBTConsensus(req *ensweb.Request, did string, qdc didcrypto
 	wt := sc.GetTransTokenInfo()
 
 	for i := range wt {
+		var tknType int
+		if wt[i].TokenValue == 1.0 {
+			tknType = token.RBTTokenType
+		} else {
+			tknType = token.PartTokenType
+		}
+
+		if c.testNet && wt[i].TokenValue == 1.0 {
+			tknType = token.TestTokenType
+		} else {
+			tknType = token.TestPartTokenType
+		}
```

Comment:
@vaishnav-ch the token type for the token that is is selected in the ith iteration can be fetched using wt[i].TokenType . 

So there is no logical need for this if conditions 

try changing the code and test

Reply:
Removed `tknType` now fetching using `wt[i].TokenType`